packages/okta-vue/src/Auth.spec.js

```diff
@@ -17,11 +22,26 @@ describe('Auth', () => {
   })
   test('sets the right user agent on AuthJS', () => {
     const expectedUserAgent = `${pkg.name}/${pkg.version} foo`
-    createLocalVue().use(Auth, {
+    const localVue = createLocalVue()
+    localVue.use(Auth, {
       issuer: '1',
       client_id: '2',
       redirect_uri: '3'
     })
     expect(mockAuthJsInstance.userAgent).toMatch(expectedUserAgent)
   })
+  test('sets the right scope and response_type when redirecting to Okta', () => {
+    const localVue = createLocalVue()
+    localVue.use(Auth, {
+      issuer: '1',
+      client_id: '2',
+      redirect_uri: '3',
+      scope: 'foo bar',
+      response_type: 'token'
+    })
+    localVue.prototype.$auth.loginRedirect()
+    const mockCallValues = mockAuthJsInstance.token.getWithRedirect.mock.calls[0][0]
+    expect(mockCallValues.responseType).toEqual(expect.arrayContaining(['token']))
+    expect(mockCallValues.scopes).toEqual(expect.arrayContaining(['foo', 'bar']))
+  })
```

Comment:
👍 awesome!  I think we should also add another test that asserts that `response_type` will default to `['id_token', 'token']` if not overridden by config, because this is a library decision that we are explicitly making.